From f10a522544145b1bf3535888c810a9f276259413 Mon Sep 17 00:00:00 2001
From: John Vincent <john.vincent.xa.bp.renesas.com>
Date: Mon, 14 Aug 2023 16:06:39 +0100
Subject: [PATCH 5/6] smarc rzg2l randomize ethernet address to avoid null

Set CONFIG_NET_RANDOM_ETHADDR=y to avoid NULL ethernet address.

Signed-off-by: John Vincent <john.vincent.xa.bp.renesas.com>
---
 configs/smarc-rzg2l_defconfig | 1 +
 1 file changed, 1 insertion(+)

diff --git a/configs/smarc-rzg2l_defconfig b/configs/smarc-rzg2l_defconfig
index 0366542b82..a103184af6 100644
--- a/configs/smarc-rzg2l_defconfig
+++ b/configs/smarc-rzg2l_defconfig
@@ -127,3 +127,4 @@ CONFIG_ISO_PARTITION=y
 CONFIG_EFI_PARTITION=y
 CONFIG_PARTITION_UUIDS=y
 CONFIG_DISTRO_DEFAULTS=y
+CONFIG_NET_RANDOM_ETHADDR=y
-- 
2.25.1

